pkg/docker/docker-entrypoint.sh

```diff
@@ -1,12 +1,12 @@
-#!/usr/bin/env sh
+#!/bin/sh
 
 set -e
 
 curl_put()
 {
     RET=$(/usr/bin/curl -s -w '%{http_code}' -X PUT --data-binary @$1 --unix-socket /var/run/control.unit.sock http://localhost/$2)
-    RET_BODY=$(printf '%s' "$RET" | cut -c 1)
-    RET_STATUS=$(echo $RET | /usr/bin/tail -c 4)
+    RET_BODY=$(echo "$RET" | sed 's/.\{3\}$//')
+    RET_STATUS=$(echo "$RET" | tail -1 | grep -ohe '...$')
```

Comment:
I think `tail -1` is an extension (I didn't even know about it), and is not documented in coreutils.

POSIX only guarantees `-n 1`.

Reply:
done